pkg/storaged/fsys-tab.jsx

```diff
@@ -549,6 +599,10 @@ export class FilesystemTab extends React.Component {
             }
             if (opt_ro)
                 opts.push("ro");
+            if (opt_nofail)
+                opts.push("nofail");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221115-130151-a219e579-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)